workbench-core/example/infrastructure/integration-tests/tests/dynamicAuthorization/group.test.ts

```diff
@@ -289,4 +290,21 @@ describe('dynamic authorization group integration tests', () => {
       }
     );
   });
+
+  describe('deleteGroup', () => {
+    it('deletes existing group', async () => {
+      const {
+        data: { groupId }
+      } = await adminSession.resources.groups.create();
+      const group = adminSession.resources.groups.group(groupId);
+
+      expect(() => group.delete()).not.toThrow();
```

Comment:
we can check that the response is what we expect now that we're returning something

Reply:
`delete` method is derived from `Resource` which has return type of `Promise<void>`. I can't declare same method name with different return type so other option would be to have secondary `delete` functionality but that feels confusing.

Reply:
lets change the default `delete` to return the promise. It wont change any existing tests' behavior, and will allow us to start testing delete response bodies moving forward